packages/integrations/react/README.md

```diff
@@ -61,6 +61,44 @@ To use your first React component in Astro, head to our [UI framework documentat
 - 💧 client-side hydration options, and
 - 🤝 opportunities to mix and nest frameworks together
 
+## Options
+
+### Children parsing
+
+When you pass children into a React component from an Astro component you only see a single child on the React side:
+
+```astro
+---
+import ReactComponent from './ReactComponent';
+---
+
+<ReactComponent>
+  <div>one</div>
+  <div>two</div>
+</ReactComponent>
+```
+
+This is because the children being passed to *are not* React nodes. Astro treats its templates as plain strings.
+
+If you are using a library that *expects* to be passed more than 1 child element; for example so that it can slot certain elements in different places, you might find this to be a blocker. You can set the experimental flag `experimentalReactChildren` to tell Astro to always pass children to React as React vnodes.
+
+There is some runtime cost to this, but it can help with compatibility. You can enable this option with:
```

Comment:
```suggestion
If you are using a library that *expects* more than one child element element to be passed, for example so that it can slot certain elements in different places, you might find this to be a blocker.

You can set the experimental flag `experimentalReactChildren` to tell Astro to always pass children to React as React vnodes. There is some runtime cost to this, but it can help with compatibility. 

You can enable this option in the configuration for the React integration:
```
Just separated out the ideas a little differently.